tests/css_case/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -292,17 +300,17 @@ a[href="KeepAttrValue"]:hover::first-letter,
 }
 
 @keyframes KeepAnimationName {
-  from {
+  FROM {
```

Comment:
`from` and `to` could technically be lowercased in `@keyframes`, but I don’t think it is worth the trouble.